tests/test_qc_plots.py

```diff
@@ -0,0 +1,118 @@
+import pytest
+import os
+from PIL import Image
+
+from tests.fixtures import *
+import cellforest as cf
+
```

Comment:
Only skimmed this file, but it looks like you did a really good job thoroughly testing